config/routes.rb

```diff
@@ -84,6 +84,11 @@
     resources :training_requests, only: [:index, :destroy] if SettingsHelper.feature_on?(:training_requests)
 
     resources :instruments do
+      if SettingsHelper.feature_on?(:walkup_reservations)
```

Comment:
I think we can just flag the link - there's not much risk of folks using this without the QR codes